Model/Behavior/WhoDidItBehavior.php

```diff
@@ -137,7 +137,7 @@ public function beforeSave(Model $Model, $options = []) {
 			$data[$field] = false;
 		}
 
-		if (!$Model->exists()) {
+		if (!$Model->exists($Model->getID())) {
```

Comment:
ID is uppercase?

Reply:
Yes. https://api.cakephp.org/2.10/class-Model.html#_getID